customer-commons/src/test/java/no/unit/nva/customer/model/CustomerTest.java

```diff
@@ -30,9 +34,12 @@ public void customerMappedToJsonAndBack() throws JsonProcessingException {
     }
 
     @Test
-    public void customerMapperCanMapBetweenCustomerDtoAndCustomerDb() {
+    public void customerMapperCanMapBetweenCustomerDtoAndCustomerDb() throws JsonProcessingException {
         CustomerDb customerDb = createCustomerDb();
         CustomerDto customerDto = customerMapper.toCustomerDto(customerDb);
+
+        System.out.println(objectMapper.writeValueAsString(customerDto));
```

Comment:
System.out?

Reply:
Good catch, not supposed to be checked in.